tests/fv3_conf/gfs_v15_run.IN

```diff
@@ -20,4 +20,5 @@ cp    @[INPUTDATA_ROOT]/FV3_input_data/*configure .
 
 if [ $DO_RRTMGP = .T. ]; then
 cp    @[INPUTDATA_ROOT]/FV3_input_data_RRTMGP/* .
+cp /scratch2/BMC/ome/Dustin.Swales/UFS/ufs-weather-model-2/FV3/ccpp/physics/physics/rte-rrtmgp/rrtmgp/data/* .
```

Comment:
We need to create a new input data set and you need to update the input data set date tag on `rt.sh`. Let's use the same date for the new input data set and the new baseline, 20210428.

Then update this line.

Reply:
@dustinswales you can remove all occurrences of `cp /scratch2/BMC/ome/...` - Dusan and I copied the contents of that directory into `@[INPUTDATA_ROOT]/FV3_input_data_RRTMGP/` for the new input data set 20210428 on all tier-1 platforms.

Reply:
Done.